bin/config-generate

```diff
@@ -1,9 +1,40 @@
-#!/bin/bash
-# Generate Odoo server configuration from templates
-set -e
-src="/opt/odoo/common/conf.d/* /opt/odoo/custom/conf.d/*"
-log INFO Merging $(ls $src | wc -l) configuration files in $OPENERP_SERVER
-conf=$(cat $src | envsubst)
-log DEBUG "Resulting configuration:
-$conf"
-echo "$conf" > $OPENERP_SERVER
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Generate Odoo server configuration from templates"""
+
+import os
+from doodbalib import logger
+from io import StringIO
+from string import Template
+
+try:
+    from configparser import RawConfigParser
+    parser = RawConfigParser(strict=False)
+except ImportError:
+    # Python 2, where strict=True doesn't exist
+    from ConfigParser import RawConfigParser
+    parser = RawConfigParser()
+
+TARGET_FILE = os.environ["OPENERP_SERVER"]
```

Comment:
Isn't another variable in new versions (like `ODOO_SERVER`)? If so I think we should fallback to that one for the day that Odoo decides to remove the deprecated variable name for not having to make any change.

Reply:
They added `ODOO_RC` in v10. I was using this one because it's the one compatible with all supported versions, but let me add support for the other one, yes.

Reply:
Oh wait, the problem is that if I add support here, I'll have to check the current odoo version to know if I should support that new variable or not (wether it's in v8/v9 or it's newer). Is it worth it, knowing this one works always? :thinking: